readme.md

```diff
@@ -39,63 +39,64 @@ $ npm -v
 * **Question:** What is the difference between forking and cloning a repository as opposed to just cloning a repository?
 
 * **Your Answer:**
-
+cloning creates a local copy associated with the original repository; forking creates a branch of the orginal repositiry
```

Comment:
Careful with the term "branching" as that applies to other actions related to git/GitHub.  :)